sdk/keyvault/keyvault-certificates/src/identifier.ts

```diff
@@ -39,26 +32,29 @@ export interface ParsedKeyVaultCertificatesIdentifier {
 }
 
 /**
- * Parses a KeyVaultIdentifier from a string URI.
+ * Parses the given Key Vault Certificate Id.
+ * Key Vault Ids come in the shape of URIs, for example
+ * a KeyVaultCertificate will have an `id` property, with a value similar to the following one:
+ *
+ *   https://<keyvault-name>.vault.azure.net/certificates/<certificate-name>/<unique-version-id>
+ *
+ * This URI can be parsed into segments using this function. The previous example would result in:
```

Comment:
```suggestion
 * Parses the given Key Vault Certificate Id which is URL that uniquely identifies 
 * a Key Vault Certificate. An example for Id is:
 *
 *   https://<keyvault-name>.vault.azure.net/certificates/<certificate-name>/<unique-version-id>
 *
 * On parsing the above Id, this function returns 
```

Reply:
`Id which is URL that uniquely identifies` seems very redundant.

What about this?

```
Parses the given Key Vault Certificate Id. An example is:
```

I like the rest of that feedback. I'll assume this approach I'm mentioning is correct for now, so I can push something. I can follow up once I receive more feedback.